14_functions_lab/04.py

```diff
@@ -1,6 +1,23 @@
-"""
-Write a function that takes minlen and
-a list of words, and returns only the words
-longer than minlen
-"""
-
+#!/usr/bin/env python
+
+__author__ = 'Eabel'
+
+import sys
+import os
+import numbers
+
+
+def longer_than(size,*words):
+    """
+    Args:
+        size (int): size of the min word length
+        words (string list): list of words
+    Returns:
+        int: subset of 'words' list that are longer then 'size'
+    """
+    if not all(isinstance(item, basestring) for item in words) or not isinstance(size, numbers.Number):

```

Comment:
Do you really need this validation? The list comprehension will explode for inputs that don't have `len()`, and without the validation you get the extra functionality that the same code can now check other things with `len`